src/ophyd_async/epics/motion/motor.py

```diff
@@ -19,7 +22,49 @@
 from ..signal.signal import epics_signal_r, epics_signal_rw, epics_signal_x
 
 
-class Motor(StandardReadable, Movable, Stoppable):
+class MotorLimitsException(Exception):
+    pass
+
+
+class InvalidFlyMotorException(Exception):
+    pass
+
+
+DEFAULT_MOTOR_FLY_TIMEOUT = 60
+DEFAULT_WATCHER_UPDATE_FREQUENCY = 0.2
+
+
+class FlyMotorInfo(BaseModel):
+    """Minimal set of information required to fly a motor:
+
+    start_position : float
```

Comment:
Please can we put these as `#:` comments above the field so they render in the docs, also please check they render in the docs!